msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/ltw/TestCase2572249.java

```diff
@@ -0,0 +1,142 @@
+import org.junit.Test;
+
+//  Updated LTW, Updated Auth app and uninstall LTW
+// https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2572249
+@LTWTests
+@RunOnAPI29Minus
+@SupportedBrokers(brokers = {BrokerLTW.class})
+public class TestCase2572249 extends AbstractMsalBrokerTest {
+
+    @Test
+    public void test_2572249() throws Throwable {
+        final String username = mLabAccount.getUsername();
+        final String password = mLabAccount.getPassword();
+
+        // install updated auth app
+        final BrokerMicrosoftAuthenticator brokerMicrosoftAuthenticator = new BrokerMicrosoftAuthenticator();
+        brokerMicrosoftAuthenticator.install();
+
+        // acquire token interactively in MsalTestApp
+        final MsalTestApp msalTestApp = new MsalTestApp();
+        msalTestApp.install();
+        msalTestApp.launch();
+        msalTestApp.handleFirstRun();
+
+        final MicrosoftStsPromptHandlerParameters promptHandlerParametersMsal = MicrosoftStsPromptHandlerParameters.builder()
+                .prompt(PromptParameter.SELECT_ACCOUNT)
+                .loginHint(username)
+                .sessionExpected(false)
+                .broker(mBroker)
+                .expectingBrokerAccountChooserActivity(false)
+                .expectingProvidedAccountInBroker(false)
+                .expectingLoginPageAccountPicker(false)
+                .expectingProvidedAccountInCookie(false)
+                .consentPageExpected(false)
+                .passwordPageExpected(true)
+                .speedBumpExpected(false)
+                .registerPageExpected(false)
+                .enrollPageExpected(false)
+                .staySignedInPageExpected(false)
+                .verifyYourIdentityPageExpected(false)
+                .howWouldYouLikeToSignInExpected(false)
+                .build();
+
+        String tokenMsal = msalTestApp.acquireToken(username, password, promptHandlerParametersMsal, true);
+        Assert.assertNotNull(tokenMsal);
+
+        // uninstall LTW
+        mBroker.uninstall();
+
+        // install OneAuthTestApp
+        final OneAuthTestApp oneAuthTestApp = new OneAuthTestApp();
+        oneAuthTestApp.install();
+        oneAuthTestApp.launch();
+        oneAuthTestApp.handleFirstRun();
+
+        // sign in to OneAuthTestApp
+        // should not prompt for password
+        final FirstPartyAppPromptHandlerParameters promptHandlerParametersOneAuth = FirstPartyAppPromptHandlerParameters.builder()
+                .broker(mBroker)
+                .prompt(PromptParameter.LOGIN)
+                .loginHint(username)
+                .passwordPageExpected(false)
+                .consentPageExpected(false)
+                .speedBumpExpected(false)
+                .sessionExpected(false)
+                .expectingBrokerAccountChooserActivity(false)
+                .expectingLoginPageAccountPicker(false)
+                .enrollPageExpected(false)
+                .build();
```

Comment:
Same as test above.

Reply:
I've changed to use the latest handleSignInWithoutPrompt method of OneAuthTestApp to deal with the situation where the password input page is not expected.